src/module.ts

```diff
@@ -120,14 +128,23 @@ export default defineNuxtModule<ModuleOptions>({
       resolve('./runtime/registry'),
     ])
 
+    addImports({
```

Comment:
i'd prefer this to be an explicit import as end users wouldn't be using it, there's already an alias you can import from `#nuxt-scripts-utils`